zkevm-circuits/src/mpt_circuit.rs

```diff
@@ -180,9 +181,14 @@ impl<F: Field> MPTConfig<F> {
         memory.allocate(meta, parent_memory(true));
         memory.allocate(meta, main_memory());
 
-        let ctx = MPTContext {
+        let mut cb = MPTConstraintBuilder::new(33 + 10, None, power_of_randomness.clone());
```

Comment:
```suggestion
        let mut cb = MPTConstraintBuilder::new(9, None, power_of_randomness.clone());
```

This should be a good test so see if the expression splitting (and CachedRegion) is working correctly! (you may need to allocate some extra columns to the cell manager because this may need quite a few extra cells to store intermediate results).